distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.adoc

```diff
@@ -0,0 +1,19 @@
+[id="distr-tracing-deploying-otel"]
+= Configuring and deploying distributed tracing data collection
+include::modules/distr-tracing-document-attributes.adoc[]
+:context: deploying-data-collection
+
+toc::[]
+
+The {OTELName} Operator uses a custom resource definition (CRD) file that defines the architecture and configuration settings to be used when creating and deploying the {OTELShortName} resources. You can either install the default configuration or modify the file to better suit your business requirements.
+
+[IMPORTANT]
```

Comment:
We have a technology-preview.adoc module to use instead of copy/pasting all of this text. If you search for that module name, you should find how to pass in the feature name, etc.